cnv/cnv_users_guide/cnv-controlling-vm-states.adoc

```diff
@@ -0,0 +1,21 @@
+[id="cnv-controlling-vm-states"]
+= Controlling virtual machines states
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-controlling-vm-states
+toc::[]
+
+With {ProductName}, you can stop, start, and restart virtual machines (VMs) from
```

Comment:
I have vague memories of being told we wanted to keep it spelt out as 'virtual machine'. It was last year so the situation might have changed but I'd probably drop the shortnames.